src/sharedData/components/SharedDataEntryBase.js

```diff
@@ -169,16 +169,33 @@ const SharedDataEntryBase = props => {
         container
         spacing={1}
         alignItems="center"
-        sx={{ fontSize: 14, color: 'gray.dark1', p: 1 }}
+        sx={{
+          fontSize: 14,
+          color: 'gray.dark1',
+          p: 1,
+        }}
       >
         <Grid
           item
-          xs={isEditingName ? 12 : 8}
+          xs={isEditingName ? 12 : 7}
           md={5}
           order={{ xs: 2, md: 2 }}
           component={StackRow}
+          sx={{
+            alignItems: {
+              xs: 'flex-start',
+              md: 'inherit',
+            },
+          }}
         >
-          <EntryTypeIcon resourceType={dataEntry.resourceType} />
+          <EntryTypeIcon
+            resourceType={dataEntry.resourceType}
+            styleProps={{
+              marginTop: { xs: '0.8em', md: 'inherit' },
+              marginLeft: { xs: '-5px', md: 'inherit' },
+              paddingRight: { xs: '5px', md: 'inherit' },
```

Comment:
Suggestion/Nitpick: Move this styling to SharedDataEntryFile.js

SharedDataEntryBase.js is used for both shared files and links. 
So if the link icon styling is in SharedDataEntryLink.js because it requires different styling than the shared files icon, it could be more consistent to have this be in the code that is specific to files.

Reply:
I'm not suggesting combining them, I'm suggesting moving the styling that is specific to the **FileIcon** to the SharedDateEntry**File**.js file